api/namex/resources/payment/payment.py

```diff
@@ -388,7 +388,7 @@ def post(self, nr_id, payment_action=NameRequestActions.CREATE.value):
                                 msg = f'Extend NR for payment.id={payment.id} nr_model.state{nr_model.stateCd}, nr_model.expires:{nr_model.expirationDate}'
                                 current_app.logger.debug(msg)
 
-                            nr_model.expirationDate = nr_model.expirationDate + timedelta(days=NAME_REQUEST_LIFESPAN_DAYS)
+                            nr_model.expirationDate = nr_svc.create_expiry_date(nr_model.expirationDate, NAME_REQUEST_LIFESPAN_DAYS)
```

Comment:
That was easy :)

Now how does this get tested?

Reply:
Sorry, I tried to create a test for the request but the tests in namex are really complicate to me and I got blocked by authentication.